### DISCUSSION TITLE
Sent mail address format  unprofessional - UPDATE

### DISCUSSION THREAD

**Discussion by LucienBourrichon** [elementary/mail#981] [Q&A]
Dear Mail developers,

Sorry if this is not the right "way of doing things" but I am far from the developers world, I am just an intensive user of your software and I feel something is missing. 

I can't seem to configure Mail so it sends out mails with the  (I though standard) "User Name" <something@domain.xyz"> format for the reply. Just the dry mail address alone looks unprofessional in receiver's mailboxes.

![image](https://github.com/elementary/mail/assets/163507890/02f6c9d6-e1d3-4a23-bbcc-bd0e001c24df)

Isn't it possible to somewhere concatenate your Real Name, which you have to provide during configuration, and your mail address when it goes out ?  Or is there a specific reason why it is implemented like this ? Or is it possible and I (along with many other people I talked to) couldn't find it easily, in which case your product would benefit from advertising the feature and telling how to do. 

I can tell you that this issue is refraining many people from dropping Evolution Mail completely under Elementary OS. 

Thanks for reading, and for your great software. 

Cheers,

Lucien.

**Comment by LucienBourrichon** [comment]
I found a solution. 

I installed Geary. 
It lacks a bit of the Elementary OS integration (with the "online accounts") but the mails being sent have the "User Name" <something@domain.xyz> format in the From field, and appear as such in the inbox of any mail reader at the receiver end. 
This looks more professional than the dry something@domain.xyz format. 

Geary is more feature rich too. I suppose it was a choice from Elementary to keep their Mail ultra simplistic. I can understand that. Geary works better for me though.  

best,

Lucien.

**Comment by LucienBourrichon** [comment]
Dear Mail developers,

I just installed (and paid for) Elementary OS 8.01. I was hoping the latest version would not have this issue. 
And yet again, when I send an e-mail, the sender address is only the unprofessional "something@domain.xyz" instead of "User Name" <something@domain.xyz>. 
Now, I looked up a bit of the proper terminology in the RFC 5322 to make this message more clear, maybe that is why never anyone bothered answering. 
So what I would like is the "display-name" field to be added to the "angle-addr" field when sending the mail, so that the receiver sees a complete "name-addr". Currently, look at the picture in my first post, what is seen on the receiver end is only the "addr-spec". 
I read that this concatenation is not supposed to happen on the mail server, but on the client. And I don't seem to find where to configure this. 

Hope this is more clear. 

Lucien.

**Reply by teamcons** [reply]
Dear Lucien

I see what you mean. I think this is more because none of the contributors sat down to work on this one particularly

Maybe if i have time... But my own contributions are on a hobby basis and there is a lot to do in other parts of the OS

Though you give a couple clues to speed this one up
